build.sbt

```diff
@@ -6,7 +6,7 @@ organization in Global := "edu.gemini.ocs"
 
 ocsVersion in ThisBuild := OcsVersion("2016A", true, 1, 1, 1)
 
-pitVersion in ThisBuild := OcsVersion("2016A", false, 1, 1, 0)
+pitVersion in ThisBuild := OcsVersion("2016B", true, 2, 1, 0)
```

Comment:
Historically we have made it so that the PIT is [Year].[Semester].[Minor] where Semester is either 1 or 2. In this case being 2016B we make it 2016B.2.1. We may review this policy for this round